src/api/ledger/parse/utils.js

```diff
@@ -4,6 +4,15 @@ const _ = require('lodash');
 const transactionParser = require('ripple-lib-transactionparser');
 const toTimestamp = require('../../../core/utils').toTimestamp;
 const utils = require('../utils');
+const BigNumber = require('bignumber.js');
+
+function adjustQualityForXRP(quality: string, takerGetsCurrency: string,
+    takerPaysCurrency: string) {
+  const shift = (takerGetsCurrency === 'XRP' ? 6 : 0)
```

Comment:
Hrmm, it seems I was thinking of this: `https://www.bitstamp.net/api/transactions/` when I recalled a "type" 

It is interesting to look at that schema though:
```
date - unix timestamp date and time
tid - transaction id
price - BTC price
amount - BTC amount
type - buy or sell (0 - buy; 1 - sell)
```
That's certainly simpler:
* How much do you want to buy or sell? (amount)
* How much will be paid for each? (price)

We aren't trying to smooth/wash over something that's just fundamentally different?

> Traditional financial markets just have conventions to not use one of the two books

Do you have any examples I could look at?